f5_os_test/heat_client_utils.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2015-2016 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+
+import pytest
+
+
+def get_template_file(template_file):
```

Comment:
The return value of this function is a series of bytes (depending on what was in the file it might be a string), it should be renamed to something like  or 'get_file_contents' (it shouldn't include the word 'template' unless there's something template-specific about it.   
The return variable ought not to assume "_str".